src/extensions/editable/bootstrap-table-editable.js

```diff
@@ -114,7 +110,7 @@ $.BootstrapTable = class extends $.BootstrapTable {
       }
 
       const data = this.getData({ escape: true })
-      const $field = this.$body.find(`a[data-name="${column.field}"]`)
+      const $field = this.$body.find('a[data-name="' + column.field + '"]')
```

Comment:
Same as in line 54

Reply:
reverted to original in this line. (Don't remember why I downgraded syntax to ES5, this was half year ago.)